training/src/app/modules/chuck-norris/components/joke-list/joke-list.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="mat-elevation-z8">
+    <table mat-table *ngIf="jokeList.data.length !== 0" [dataSource]="jokeList" multiTemplateDataRows>
```

Comment:
in Angular, false conditions are: false, undefined, null, 0
in this case u can just say: "jokeList.data.length" - this will give u a number.
if its 0 that means its false, otherwise its true